apps/web/app/components/providers/wallet-provider.tsx

```diff
@@ -5,8 +5,15 @@ import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { type ReactNode, useMemo } from "react";
 import { type State, WagmiProvider, cookieStorage, createStorage } from "wagmi";
 import { mainnet, sepolia } from "wagmi/chains";
+import { structuralSharing } from "wagmi/query";
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      structuralSharing,
+    },
+  },
+});
```

Comment:
It seems this is no longer necessary according to: https://github.com/wevm/wagmi/issues/4233#issuecomment-2337225304. Testing it now.

Reply:
:clap:

Reply:
Works, pushing changes.